### ISSUE TITLE
(Material) FormlySliderTypeComponent is not exported

### ISSUE THREAD

**Issue by wadefleming-nz** [ngx-formly/ngx-formly#3094]
**Description**
I want to extend `FormlySliderTypeComponent`, however I cannot import it to my project.

It is not exported like other similar types ie: `\src\material\select\src\public_api.ts`:

```
export { FormlyMatSelectModule } from './select.module';
export { FormlyFieldSelect } from './select.type';     // << ------- type is exported
```

whereas `\src\material\slider\src\public_api.ts` has only:
```
export { FormlyMatSliderModule } from './slider.module';
```

**Minimal Reproduction**

Extend a type from `FormlySliderTypeComponent` as shown here: https://stackblitz.com/edit/ngx-formly-ui-material-49pves?file=src%2Ftypes%2Fslider-extended.type.ts

Note that `FormlySliderTypeComponent` auto imports from `@ngx-formly/material/slider/slider.type` which doesn't work as its not public, hence the app fails to compile.

**Your Environment**

- Angular version: 13.0.0
- Formly version: 5.10.26

**Additional Info**

I noticed that `FormlyDatepickerTypeComponent` and `FormlyToggleTypeComponent` are also not exported, and that all of the not-imported types have the `Component` suffix. Maybe there is something else going on here?

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.10.27](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.27) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by wadefleming-nz** [comment]
Wow, appreciate this being addressed so quickly!

I don't need them now but as above `FormlyDatepickerTypeComponent ` and `FormlyToggleTypeComponent`  area also similarly not exported and have the `Component` suffix. You may want to consider doing the same for them.

Thanks

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.10.28](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.28) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
